data/json/items/armor/brigandine

```diff
@@ -0,0 +1,320 @@
+[
+ {
+    "id": "armor_lc_brigandine",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "brigandine with mild steel plates" },
+    "description": "",
+    "weight": "7000 g",
+    "volume": "2750 ml",
+    "price": 60000,
+    "price_postapoc": 12000,
+    "to_hit": -5,
+    "material": [ "lc_steel", "leather" ],
+    "bashing": 8,
+    "symbol": "[",
+    "looks_like": "armor_larmor",
+    "color": "light_gray",
+    "warmth": 20,
+    "longest_side": "60 cm",
+    "material_thickness": 3.1,
+    "flags": [ "OUTER", "STURDY", "CONDUCTIVE", "ONLY_ONE" ],
```

Comment:
Shouldn't this being rigid eliminate need for ONLY_ONE 🤔

Reply:
probably not, i just copied it from sheet metal chest guard and don't adjust all stats curantly